sql-plugin/src/main/311until320-nondb/scala/com/nvidia/spark/rapids/shims/v2/SparkBaseShims.scala

```diff
@@ -221,8 +221,7 @@ abstract class SparkBaseShims extends Spark31XShims with Logging {
     GpuOverrides.expr[Abs](
       "Absolute value",
       ExprChecks.unaryProjectAndAstInputMatchesOutput(
-        TypeSig.implicitCastsAstTypes, TypeSig.gpuNumeric + TypeSig.DECIMAL_128_FULL,
-        TypeSig.numeric),
+        TypeSig.implicitCastsAstTypes, TypeSig.numeric, TypeSig.numeric),
```

Comment:
`TypeSig.numeric` is not supposed to be used on GPU type checks, it is for CPU type checks. That is the reason we were doing `TypeSig.gpuNumeric + TypeSig.DECIMAL_128_FULL`.